src/test/java/bio/terra/service/snapshotbuilder/SnapshotRequestDaoTest.java

```diff
@@ -67,13 +72,24 @@ private SnapshotAccessRequestModel createRequest(SnapshotAccessRequest snapshotA
   }
 
   private void verifyResponseContents(SnapshotAccessRequestModel response) {
+    verifyResponseContents(response, List.of());
+  }
+
+  private void verifyResponseContents(
+      SnapshotAccessRequestModel response, List<String> ignoredFields) {
     SnapshotAccessRequestModel expected =
         SnapshotBuilderTestData.createSnapshotAccessRequestModel(sourceSnapshot.getId());
 
+    // always ignore id and createdDate which are unique
+    List<String> allIgnoredFields = new ArrayList<>();
+    allIgnoredFields.add("id");
+    allIgnoredFields.add("createdDate");
+    allIgnoredFields.addAll(ignoredFields);
+
     assertThat(
         "Given response is the same as expected.",
         response,
-        samePropertyValuesAs(expected, "id", "createdDate", "flightid", "createdSnapshotId"));
+        samePropertyValuesAs(expected, allIgnoredFields.toArray(new String[0])));
```

Comment:
OK, I now remember this code. You could instead keep the data in an array. It's about the same amount of code as using a `List` and you don't have to copy it back to an array at the end.
```
    String[] ignoredFieldsArray = new String[ignoredFields.length + 2];
    System.arraycopy(ignoredFields, 0, ignoredFieldsArray, 0, ignoredFields.length);
    ignoredFieldsArray[ignoredFields.length] = "id";
    ignoredFieldsArray[ignoredFields.length + 1] = "createdDate";
```

Reply:
This is a bit better. Still need to round-trip from arrays to Stream and back, but it's not much code:
```
    String[] ignored = Stream.concat(Arrays.stream(ignoredFields), Stream.of("id", "createdDate"))
        .toArray(String[]::new);
```